packages/experiments/src/components/Toggle/Toggle.types.ts

```diff
@@ -88,39 +99,46 @@ export type IToggleViewProps = Pick<IToggleProps, 'as' | 'label' | 'ariaLabel' |
      * Text to display next to the toggle.
      */
     text?: string;
+
+    /**
+     * Reference to the toggle button.
+     */
+    toggleButton?: React.RefObject<HTMLButtonElement>;
   };
 
 /**
  * Styles for the Toggle component.
  */
-export interface IToggleStyles {
-  /**
-   * Root element.
-   */
-  root: IStyle;
-
-  /**
-   * Label element above the toggle.
-   */
-  label: IStyle;
-
-  /**
-   * Container for the toggle pill and the text next to it.
-   */
-  container: IStyle;
-
-  /**
-   * Pill, rendered as a button.
-   */
-  pill: IStyle;
+export interface IToggleStyleVariablesTypes {
+  pillBackground?: string;
+  pillHoveredBackground?: string;
+  pillBorderColor?: string;
+  pillHoveredBorderColor?: string;
+  pillJustifyContent?:
+    | 'initial'
+    | 'inherit'
+    | 'unset'
+    | 'flex-start'
+    | 'flex-end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'stretch'
+    | undefined;
```

Comment:
These values seem like they should be defined somewhere already, or centralized if not...